lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_auth_client.rb

```diff
@@ -83,8 +83,11 @@ def service_account(google_service_path)
         service_account_credentials.fetch_access_token!["access_token"]
       rescue Errno::ENOENT
         UI.user_error!("#{ErrorMessage::SERVICE_CREDENTIALS_NOT_FOUND}: #{google_service_path}")
-      rescue Signet::AuthorizationError
-        UI.user_error!("#{ErrorMessage::SERVICE_CREDENTIALS_ERROR}: #{google_service_path}")
+      rescue Signet::AuthorizationError => error
+        UI.user_error!(
```

Comment:
Should we only display this when `debug` is set to `true`?

Reply:
If this call fails I suspect it's always helpful to have more info. If we were showing details about successful requests here too, I think that would make sense to show in debug mode only. FWIW I looked for a "debug" option in the Google auth library but didn't find anything like that.

Reply:
Gotcha. Should we consider showing messages if any of the steps fail? Or is there a particular reason it will be useful to show this message even if the debug flag isn't set?

Reply:
In this case the error on its own is not helpful since there's so little information. Same with the `firebase_token` method (I added this to that one as well). But maybe since we're just dumping the response it feels more like debug level info? I'd still probably lean towards giving too much info vs too little in the case of an error though.

Reply:
Gotcha. I agree probably want to bias showing too much info vs. too little. Are we sure the message won't have any sensitive information? There's an understanding that when the debug flag is set we may show sensitive information, but that's not necessarily the case when it is not set. If we think this won't display any sensitive information LGTM 🙂

Reply:
That's a good point. It probably won't, but seems better to just require the debug flag. I think that will be most folks first instinct anyway when they're seeing an error they don't understand.

Updated.